src/actions/actions.test.js

```diff
@@ -1,41 +1,30 @@
 import animals from '../utils/data/animals.mock';
 import {
-  CHANGE_VIEW,
   RESET_DATA,
   TOGGLE_NODE_CLICKED,
-  TOGGLE_NODE_DISABLED,
   TOGGLE_NODES_DISABLED,
   TOGGLE_NODE_HOVERED,
-  TOGGLE_PARAMETERS,
   TOGGLE_TAG_ACTIVE,
   TOGGLE_TAG_FILTER,
   TOGGLE_THEME,
+  TOGGLE_TYPE_DISABLED,
   TOGGLE_TEXT_LABELS,
   UPDATE_CHART_SIZE,
-  changeView,
+  UPDATE_FONT_LOADED,
```

Comment:
Are the consts exported just for tests or used elsewhere (just a silly question)

Reply:
The consts are used in the reducer

Reply:
👍